Car.js

```diff
@@ -1,26 +1,84 @@
-function Car(make, model, year, color, seats){
+function Car(make, model, year, color, seats, passengers){
   this.make = make;
   this.model = model;
   this.year = year;
+  this.color = color;
+  this.seats = seats;
   //TODO: add color, seats here
 
   this.running = false;
   this.owner = "manufacturer";
   this.previousOwners = [];
-}
+  this.passengers = [];
```

Comment:
Here, you can also use notation to specify a default value if passengers is not passed in:
```this.passengers = passengers || [];```